sdk/core/System.ClientModel/tests/Pipeline/ClientPipelineTests.cs

```diff
@@ -311,11 +312,13 @@ public async Task RequestOptionsCanCustomizePipeline()
         List<string> observations = ObservablePolicy.GetData(message);
 
         int index = 0;
-        Assert.AreEqual(7, observations.Count);
+        Assert.AreEqual(9, observations.Count);
```

Comment:
Why 9?

Reply:
That is the number of observable policies in the pipeline.  It went from 7->9 in this PR because I wasn't previously testing that we could add policies in the `BeforeTransport` position via `RequestOptions` -- I added such a policy in this test.